tests/zfs-tests/tests/functional/casenorm/mixed_create_failure.ksh

```diff
@@ -0,0 +1,56 @@
+#!/bin/ksh -p
```

Comment:
You're going to need to add to add this test case to the `casenorm` section of the `tests/runfiles/linux.run`.  This is why it wasn't run by the automated testing.

Reply:
Ah ! I missed that.